fs/operations/operations.go

```diff
@@ -475,6 +484,12 @@ func Copy(ctx context.Context, f fs.Fs, dst fs.Object, remote string, src fs.Obj
 						if err == nil {
 							newDst = dst
 							err = closeErr
+							// Remove resume cache file (if one was created) when Put/Upload is successful
+							cacheFile := filepath.Join(config.CacheDir, "resume", f.Name(), f.Root(), remote)
```

Comment:
I think this is the incorrect path with the path mangling done when creating the cache dir?